app/views/curation_concerns/base/_form_relationships.html.erb

```diff
@@ -8,13 +8,13 @@
     <p>This Asset will be added as a representation of CITI resource <%= params.fetch(:resource_title) %>.</p>
     <%= f.hidden_field :additional_representation, value: params.fetch(:representation_for) %>
   <% end %>
-  <%= f.input :representations_for, as: :hidden_multi_select, input_html: { class: 'form-control' }, required: false %>
+  <%= f.input :representations_for, as: :hidden_multi_select, input_html: { class: 'form-control', id: 'representations_data' }, required: false %>
```

Comment:
see my other comments about DOM elements. That should remove the need for the id here.